azurerm/internal/services/apimanagement/resource_arm_api_management_api_operation.go

```diff
@@ -250,15 +250,27 @@ func expandApiManagementOperationRequestContract(input []interface{}) (*apimanag
 	}
 
 	vs := input[0].(map[string]interface{})
+	if vs == nil {
+		return nil, fmt.Errorf("Error mapping input")
```

Comment:
We typically just ignore errors like this
```suggestion
		return nil, nil
```